synapse/config/jwt.py

```diff
@@ -48,6 +50,7 @@ def read_config(self, config, **kwargs):
             self.jwt_algorithm = None
             self.jwt_issuer = None
             self.jwt_audiences = None
+            self.jwt_subject_claim = None
```

Comment:
```suggestion
            self.jwt_subject_claim = None
            self.jwt_issuer = None
            self.jwt_audiences = None
```

Reply:
(This just keeps the order the same as above for each property, which I find nice when coming back to code far in the future!)